spec/tree_spec.rb

```diff
@@ -1,14 +1,73 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(described_class.is_a? Class).to be_truthy # for some reason expect(true).to be_true is failing
```

Comment:
Every test should really only check one thing. Maybe split the aging up into multiple tests. Make sure it grows and starts producing fruit, make sure it doesn't produce fruit before a certain age, make sure it dies after a certain age, make sure it is growing.